Nelsonomotuyole.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Nelson omotuyole",
+  "displapstatsPublic": true,
+  "type": "personal",
+  "bio": "Teacher turned **programmer** | Intergrated science student from *The university of lagos Nigeria | presently Learning **phyton pro bootcamp with angela yu**  ",
+  "avatar": "https://github.com/Nelsonomotuyole.png"
+  "Links": [
```

Comment:
`Links` should be `links`

Reply:
yes how do i coorect the mistakes

Reply:
> yes how do i coorect the mistakes

You can make another commit with the changes requested on the same branch you did earlier
Did you did it in GitHub or using a code editor?

Reply:
i did it with GitHub

Reply:
> i did it with GitHub

so you do the exact same thing again with these new changes and commit again

Reply:
go to your forked repo, find your file, edit the changes and commit

Reply:
When I make changes should I commit directly to the to patch-1 branch or creat a new branch

Reply:
> When I make changes should I commit directly to the to patch-1 branch or creat a new branch

make it on the same `patch-` branch

Reply:
Okay thanks you so much really appreciate